src/content/topics/integrations/more/compass.mdx

```diff
@@ -0,0 +1,152 @@
+  <CalloutDescription>
+
+  The Compass integration is available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
+
+  </CalloutDescription>
+</Callout>
+
+## Overview
+
+This topic explains how to use the Compass integration for LaunchDarkly.
+
+<Callout intent="info">
+  <CalloutTitle>The Compass integration is managed by Atlassian</CalloutTitle>
+  <CalloutDescription>
+
+  The Compass integration was built by and is managed by Atlassian. For questions about the integration or for technical support, [contact Atlassian](https://community.atlassian.com/t5/Compass-Alpha/gh-p/compass-alpha).
+
+  </CalloutDescription>
+</Callout>
+
+The Compass integration for LaunchDarkly displays a timeline of feature flag changes related to a Compass component, as well as a complete list of feature flags and their current state. This helps you visualize how recent changes to feature flags may be impacting component health.
+
+## Prerequisites
+
+To connect Compass to LaunchDarkly, you must meet the following prerequisites:
+
+* Be a Compass administrator on your Atlassian site.
+* Have a LaunchDarkly [API access token](/home/account-security/api-access-tokens) for use with Compass. To generate a token:
+  * Create a token with the **Reader** role.
+  * If you are on an Enterprise plan, check the **This is a service token** option when creating your access token.
+  * If you are on a Pro plan, leave this option unchecked.
+  * Be sure to copy the new token and save it somewhere secure.
+
+## Setting up the Compass integration
+
+The Compass integration is managed from the Compass product on your Atlassian Cloud site. Other than creating an API access token, there are no Compass settings or preferences to configure in LaunchDarkly.
+
+Here's how to set up your Compass integration:
+
+1. Generate a LaunchDarkly API access token as described in the [Prerequisites](#prerequisites) section.
+2. Go to your Atlassian Compass site, for example, `https://mysite.atlassian.net/compass`.
+3. In the top navigation, click the **Apps** link.
+4. Find **LaunchDarkly** in the list of apps and click **Install**.
+5. After the LaunchDakly app is installed, click **Configure**.
+6. Click **Allow access** to approve the access to your Atlassian account requested by the LaunchDarkly app.
+7. On the **LaunchDarkly is requesting Access** screen, review the requested permissions, then click **Accept**.
+
+<Callout intent="info">
+  <CalloutTitle>You must authorize the requested permissions for the integration to work</CalloutTitle>
+  <CalloutDescription>
+
+  If you click **Cancel**, the integration cannot access your Compass data.
+
+  </CalloutDescription>
+</Callout>
+
+![LaunchDarkly authorization screen for Compass.](compass-ld-integration-auth-dialog.png)
```

Comment:
```suggestion
```

Reply:
@ember-stevens I agree this image is probably unnecessary. I'll remove the png from the changeset.